<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasWidget</class>
 <widget class="QWidget" name="CanvasWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CanvasView" name="canvasView"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="MarkWidget" name="markWidget">
      <attribute name="title">
       <string>方案评估</string>
      </attribute>
     </widget>
     <widget class="AttributeWidget" name="attributeWidget">
      <attribute name="title">
       <string>作业单位属性</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MarkWidget</class>
   <extends>QWidget</extends>
   <header>OperationUnit/MarkWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CanvasView</class>
   <extends>QGraphicsView</extends>
   <header>OperationUnit/CanvasView.h</header>
  </customwidget>
  <customwidget>
   <class>AttributeWidget</class>
   <extends>QWidget</extends>
   <header>OperationUnit/AttributeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
